src/py/reactpy/reactpy/core/hooks.py

```diff
@@ -49,7 +61,9 @@ def use_state(initial_value: Callable[[], _Type]) -> State[_Type]: ...
 def use_state(initial_value: _Type) -> State[_Type]: ...
 
 
-def use_state(initial_value: _Type | Callable[[], _Type]) -> State[_Type]:
+def use_state(
+    initial_value: _Type | Callable[[], _Type], *, server_only: bool = False
+) -> State[_Type]:
```

Comment:
Issue D - `use_state` is client visible by default. `use_ref` is server only by default. Both have a `server_only` argument.